src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png

```diff
@@ -0,0 +1 @@
+../../../../img/chatzilla22.png
```

Comment:
As long as the content does not differ, I guess symlinking directories would be best.